jp/src/edu/hillel/lesson23/Factory/FurnitureFactory.java

```diff
@@ -0,0 +1,12 @@
+package edu.hillel.lesson23.Factory;
+
+public class FurnitureFactory {
```

Comment:
It's a not an AbstractFactory. You need to make it as abstract class or interface and create a couple of factory imlementations

Reply:
Changed